Makefile

```diff
@@ -64,6 +64,7 @@ unit-test-nocoverage-host: golint-host govet-host
 
 build: golint govet
 	vagrant ssh mon0 -c 'sudo -i sh -c "cd $(GUESTGOPATH); make run-build"'
+	make run
```

Comment:
oh, k. I think he might have been looking for `make demo` kind of target of netplugin which not only create a vagrant setup but also starts the netplugin/netmaster etc

But I will let him comment, otherwise sgtm.

On a unrelated note, bringing up volplugin might interfere with system-tests but I think sanity will tell us :)